amd/comgr/src/comgr-compiler.cpp

```diff
@@ -1638,7 +1789,8 @@ amd_comgr_status_t AMDGPUCompiler::linkBitcodeToBitcode() {
       // on Windows. Replace with '_'
       std::replace(OutputFileName.begin(), OutputFileName.end(), ':', '_');
 
-      std::string OutputFilePath = OutputDir.str().str() + path::get_separator().str() + OutputFileName;
```

Comment:
A piece of advice: try to avoid these kind of unrelated format changes in a pull-request. It spooks the reviewers.